v-next/hardhat/src/internal/builtin-plugins/solidity-test/reporter.ts

```diff
@@ -62,7 +62,7 @@ export async function* testReporter(
           const name = testResult.name;
           const status: TestStatus = testResult.status;
           const details = [
-            ["duration", `${testResult.durationMs} ms`],
+            ["duration", `${testResult.durationNs} ns`],
```

Comment:
We now return `ns` instead of `ms`. I replaced all usages of `ms` that I found with `ns`.